### ISSUE TITLE
How to get the log when "Please see logs of background page for more details" occurs during customizing keymap

### ISSUE THREAD

**Issue by caidwang** [gdh1995/vimium-c#37]
When I change the extension from Vimium and customize the keymap to Vimium-style, some errors raises with the warning "Please see logs of background page for more details". How can I get the logs to check the errors? Is there any suitable configuration for origin Vimium users? I try to import the configuration exported from Vimium but it doesn't work.  And I quiet wonder the reason for changing the default keymap from Vimium.

**Comment by gdh1995** [comment]
### log

On `chrome://extensions/?id=hfjbmagddngcpeloejdejnfgbamkjaeg`, turn on the `developer mode` switch at top right corner, and then click a link of "background page" in the "Inspect views" section. Then Chrome will popup a debugging window and then the "Console" panel has log information.

### configuration

There' no lists like Vimium's. But I've added a "recommended" settings file and you can click the down arrow at the right of "Import Settings". Most of its key mappings uses only your left hand.

Vimium C has a different format of exported configuration files, so the one from Vimium is not suitable.

Besides, although Vimium C supports "map key command", but it has its own command options format and then the Vimium's might cause errors.

Basically, Vimium C's command options look like:
* `map key command opt1="here-is-a-string" opt2=false opt3 opt4=[1,2,3]`
* if no `=...`, then it's treated as `optName=true`, so the value of `opt3` is `true`
* otherwise, JSON-parse the string at the right of `=`
  * if you need a string including spaces character, you can replace every space with `\u0020` (JSON escaping format of `space`)
  * tip: there should be no `space` after `,` in a JSON array

Note: The two differences above is mainly because Vimium C implements these features earlier than Vimium.

**Comment by caidwang** [comment]
Thanks a lot. Everything goes well now.

**Comment by gdh1995** [comment]
what's your typo and how did you fix it?

maybe I could learn something from your typos and add code to support them directly.
